ingestion/functions/parsing/riograndedosul/riograndedosul.py

```diff
@@ -0,0 +1,242 @@
+                "end": convert_date(confirmed),
+            },
+        }
+    ]
+    if symptoms:
+        events.append({
+            "name": "onsetSymptoms",
+            "dateRange":
+            {
+                "start": convert_date(symptoms),
+                "end": convert_date(symptoms),
+            },
+        })
+    if hospitalized != "NAO":
+        events.append({
+            "name": "hospitalAdmission",
+            "dateRange":
+            {
+                "start": convert_date(hospitalized),
+                "end": convert_date(hospitalized),
+            },
+            "value": "Yes"
+        })
+    if death:
+        events.append({
+            "name": "outcome",
+            "dateRange":
+            {
+                "start": convert_date(death),
+                "end": convert_date(death),
+            },
+            "value": "Death"
+        })
+    return events
+
+
+def convert_symptoms(fever: str, cough: str, sorethroat: str, shortbreath: str, other: str):
+    symptoms = []
+    if fever == "SIM":
+        symptoms.append("Fever")
+    if cough == "SIM":
+        symptoms.append("Cough")
+    if sorethroat == "SIM":
+        symptoms.append("Sore Throat")
+    if shortbreath == "SIM":
+        symptoms.append("Shortness of Breath")
+    if other == "SIM":
+        symptoms.append("Other")
+    return symptoms
+
+
+def convert_preexisting_conditions(raw_commorbidities: str):
+    preexistingConditions = {}
+    if not raw_commorbidities:
+        return None
+    preexistingConditions["hasPreexistingConditions"] = True
+    commorbidities_list = []
+
+    for key in _COMORBIDITIES_MAP:
+        if key in raw_commorbidities:
+            commorbidities_list.append(_COMORBIDITIES_MAP[key])
+
+    preexistingConditions["values"] = commorbidities_list
+    return preexistingConditions
+
+
+def convert_ethnicity(raw_ethnicity: str):
+    if raw_ethnicity == "Preta":
+        return "Black"
+    elif raw_ethnicity == "Parda":
+        return "Mixed"
+    elif raw_ethnicity == "Amarela":
+        return "Asian"
+    elif raw_ethnicity == "Branca":
+        return "White"
+    elif raw_ethnicity == "Indigena":
+        return "Indigenous"
+
+
+def convert_demographics(gender: str, age: str, ethnicity: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        if age == "80 e mais":
+            demo["ageRange"] = {
+                "start": 80,
+                "end": 120
+            }
+        elif age == "<1":
+            demo["ageRange"] = {
+                "start": 0,
+                "end": 0
+            }
+        else:
+            age_range = age.split(' a ')
+            demo["ageRange"] = {
+                "start": float(age_range[0]),
+                "end": float(age_range[1])
+            }
+        if ethnicity != "NAO INFORMADO":
+            demo["ethnicity"] = convert_ethnicity(ethnicity.title())
+    return demo
```

Comment:
As nit'd previously, it'd be better to return `None` if we have none of gender/age/ethnicity.